packages/turf-along/test.js

```diff
@@ -17,15 +20,31 @@ test('turf-along', t => {
     const pt7 = along(line, 100, options);
     const pt8 = along(line.geometry, 0, options);
     const fc = featureCollection([pt1, pt2, pt3, pt4, pt5, pt6, pt7, pt8]);
-
+    
     fc.features.forEach(f => {
         t.ok(f);
         t.equal(f.type, 'Feature');
         t.equal(f.geometry.type, 'Point');
```

Comment:
why these checks? what else but a point could be returned by `along`?

Reply:
They were the original tests - they are a bit pointless but I've left them